bigbluebutton-client/resources/prod/lib/kurento-extension.js

```diff
@@ -231,17 +235,26 @@ Kurento.prototype.downscaleResolution = function (oldWidth, oldHeight) {
 Kurento.prototype.init = function () {
   const self = this;
   if ('WebSocket' in window) {
-    console.log('this browser supports websockets');
+    console.debug('this browser supports websockets');
     this.ws = new WebSocket(this.socketUrl);
 
     this.ws.onmessage = this.onWSMessage.bind(this);
     this.ws.onclose = (close) => {
-      kurentoManager.exitScreenShare();
-      self.onFail('Websocket connection closed');
+      if (this.started) {
```

Comment:
Would it make sense to have a log here that says something like, "Websocket closed, this.started={this.started}". Just to track the flow.